### ISSUE TITLE
Radio Buttons appear warped in IE11

### ISSUE THREAD

**Issue by amsheehan** [material-components/material-components-web#632]
### What MDC-Web Version are you using?

0.10.0

### What browser(s) is this bug affecting?

IE11

### What OS are you using?

Windows 8.1 on VirtualBox VM

### What are the steps to reproduce the bug?

1. Open IE11
2. Navigate to [demo site](http://material-components-web.appspot.com/radio.html)

### What is the expected behavior?

Well formed and beautifully designed radio buttons

### What is the actual behavior?

Radio buttons that look like they went through the wash

### Any other information you believe would be useful?

![screen shot 2017-05-11 at 10 25 25 am](https://cloud.githubusercontent.com/assets/1578263/25954541/ab53a256-3634-11e7-9841-f781c6513d69.png)

**Comment by anton-kachurin** [comment]
http://stackoverflow.com/a/39674356

TL;DR:
1. Change [.mdc-radio flex-basis](https://github.com/material-components/material-components-web/blob/master/packages/mdc-radio/mdc-radio.scss#L43) to `auto`, e.g. `flex: 0 0 auto;`
2. Use `max-width` instead of `width` on [.mdc-radio](https://github.com/material-components/material-components-web/blob/master/packages/mdc-radio/mdc-radio.scss#L44)

Option 1 is my favourite.
